src/InstallationValidator.Core/Services/BatchOutputFileComparer.cs

```diff
@@ -70,11 +70,15 @@ private OutputComparisonResult compareOutputs(BatchSimulationComparison simulati
          return _comparisonStrategy.CompareOutputs(new BatchOutputComparison(simulationComparison1, outputValue1), new BatchOutputComparison(simulationComparison2, outputValue2), comparisonSettings);
       }
 
-      private IEnumerable<OutputComparisonResult> missingOutputsFrom(BatchSimulationComparison simulationComparison1, BatchSimulationComparison simulationComparison2, ComparisonSettings comparisonSettings)
+      private IEnumerable<OutputComparisonResult> missingOutputsFrom(
+         BatchSimulationComparison simulationComparison1, 
+         BatchSimulationComparison simulationComparison2, 
+         ComparisonSettings comparisonSettings,
+         string folderType)
       {
          return simulationComparison1.Simulation.OutputValues.Select(x => x.Path)
             .Where(p => !simulationComparison2.HasOutput(p))
-            .Select(p => new MissingOutputComparisonResult(p, comparisonSettings, simulationComparison1.Name, simulationComparison2.Name));
+            .Select(p => new MissingOutputComparisonResult(p, comparisonSettings, simulationComparison1.Name, simulationComparison2.Folder, folderType));
```

Comment:
@Yuri05 Also fixing this one here. 
I am not sure why it was using the name here but is should be folder I believe